src/model/affiliation.ts

```diff
@@ -47,6 +48,33 @@ export class Affiliation extends BackmatterEntity {
     this.institution = json.institution as { name: string } || {name: ''};
     this.address = json.address as { city: string } || {city: ''};
   }
+
+  public toXml(): Element {
+    const xmlDoc = new DOMImplementation().createDocument(null, null); 
+    const affEl = xmlDoc.createElement('aff');
+    affEl.setAttribute('id', this.id);
+  
+    const label = xmlDoc.createElement('label');
+    label.appendChild(xmlDoc.createTextNode(this.label || ''));
+    affEl.appendChild(label);
+  
+    const department = xmlDoc.createElement('institution');
+    department.appendChild(xmlDoc.createTextNode(get(this, 'institution.name', '')));
+    affEl.appendChild(department);
+  
+    const addressLine = xmlDoc.createElement('addr-line');
```

Comment:
Just cross checked this against the [spec](https://github.com/libero/editor/issues/28), can't see an example with `addr-line`. Is this something new? Might be worth running past Fred if you're not sure.

Reply:
So the example article we're working against has 
```
<aff id="aff1">
<label>1</label>
<institution content-type="dept">Biomedical and Biological Sciences Program</institution>
<institution>Cornell University</institution>
<addr-line>
<named-content content-type="city">Ithaca</named-content>
</addr-line>
<country>United States</country>
</aff>
<aff id="aff2">
<label>2</label>
<institution content-type="dept">Department of Molecular Biology and Genetics</institution>
<institution>Cornell University</institution>
<addr-line>
<named-content content-type="city">Ithaca</named-content>
</addr-line>
<country>United States</country>
</aff>
```

which is why this is how it is, I'll check with Fred to confirm

Reply:
This is the way we currently capture cities. We want Editor to use `<city>` instead of `<addr-line><name-content content-type="city">`, as in [the spec](https://github.com/libero/editor/issues/28).

The XSL I provided will do the mapping back and forth as needed.